agent/topology_forwarder.go

```diff
@@ -72,30 +74,37 @@ func (t *TopologyForwarder) OnDisconnected(c *shttp.WSAsyncClient) {
 	}
 }
 
+// OnNodeUpdated event
 func (t *TopologyForwarder) OnNodeUpdated(n *graph.Node) {
 	t.WSAsyncClientPool.SendWSMessageToMaster(shttp.NewWSMessage(graph.Namespace, graph.NodeUpdatedMsgType, n))
 }
 
+// OnNodeAdded event
 func (t *TopologyForwarder) OnNodeAdded(n *graph.Node) {
 	t.WSAsyncClientPool.SendWSMessageToMaster(shttp.NewWSMessage(graph.Namespace, graph.NodeAddedMsgType, n))
 }
 
+// OnNodeDeleted event
 func (t *TopologyForwarder) OnNodeDeleted(n *graph.Node) {
 	t.WSAsyncClientPool.SendWSMessageToMaster(shttp.NewWSMessage(graph.Namespace, graph.NodeDeletedMsgType, n))
 }
 
+// OnEdgeUpdated event
 func (t *TopologyForwarder) OnEdgeUpdated(e *graph.Edge) {
 	t.WSAsyncClientPool.SendWSMessageToMaster(shttp.NewWSMessage(graph.Namespace, graph.EdgeUpdatedMsgType, e))
 }
 
+// OnEdgeAdded event
 func (t *TopologyForwarder) OnEdgeAdded(e *graph.Edge) {
 	t.WSAsyncClientPool.SendWSMessageToMaster(shttp.NewWSMessage(graph.Namespace, graph.EdgeAddedMsgType, e))
 }
 
+// OnEdgeDeleted event
 func (t *TopologyForwarder) OnEdgeDeleted(e *graph.Edge) {
 	t.WSAsyncClientPool.SendWSMessageToMaster(shttp.NewWSMessage(graph.Namespace, graph.EdgeDeletedMsgType, e))
 }
 
+// NewTopologyForwarder is a mechanism aim to distribute all graph node notification to WebSocket client pool.
```

Comment:
NewTopologyForwarder is a mechanism aiming to distribute all graph node notifications to a WebSocket client pool